common/reflectivity/reflectivity.h

```diff
@@ -0,0 +1,22 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <deque>
+
+namespace rs2
+{
+    class reflectivity 
```

Comment:
Document, please, what this calculates, how to use, what the range of values is, etc.

Reply:
I should also add the requirement reference